homeworks/alina.yaremenko_AlinaYaremenko/<GamesProject> /style.css

```diff
@@ -0,0 +1,13 @@
+body {
+  font-family: Roboto;
```

Comment:
Please add some styles to elements on the page as stated in point 7 of the [task](https://github.com/MastersAcademy/fe-course-basic-2023/issues/61)